signer/src/block_observer.rs

```diff
@@ -20,6 +20,7 @@
 use std::collections::HashMap;
 
 use crate::error;
+use crate::stacks_api::fetch_unknown_ancestors;
```

Comment:
Nit: I feel pretty strongly against importing functions directly. Since by convention they have the same casing as modules, they are more difficult to distinguish when reading the imports. Moreover, I like to be able to assume that any function called without any explicit namespace is local to the current module.

So I would very much prefer to just import the `stacks_api` module here instead.
```suggestion
use crate::stacks_api
```

Reply:
👍🏽 makes a lot of sense, will change.

Reply:
I like this rule of thumb a lot.

Reply:
Thanks! :pray: